main.css

```diff
@@ -1,3 +1,31 @@
-h1 {
-    color: red;
+body {
+    font-size: 18px;
+    font-family: "Avenir Next", "Avenir", -apple-system, system-ui,
+    -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen, Ubuntu, Cantarell, 'Open Sans', 'Helvetica Neue', sans-serif;
+}
+
+html, 
+body {
+    height: 100%;
+}
+
+.main-content {
+    display: flex;
+    height: 50%;
+}
+
+.text-editor {
+    flex: 1;
+    padding:  30px ;
```

Comment:
No **magic numbers** in our code.

Reply:
I understand the meaning of magic numbers, However i don't understand why r we calling these magic numbers.